main.py

```diff
@@ -148,5 +149,76 @@ async def keen(context):
     await context.send(result)
 
 
-bot.run(TOKEN)
+@bot.command(name='daily', help='Will give the daily leaderboard for specified day')
+async def daily(context, day : str = str((datetime.datetime.today() - datetime.timedelta(hours=5)).day)):
+    # The default input is whatever day's competition has just come out.  SO at 2AM will still show previous day's leaderboard
```

Comment:
Typo "SO" -> "So"
Probably say "default day" instead of "default input"
Maybe say "challenge" instead of "competition"? I'd say a competition is a series of challenges.
I'd also use 4:59AM UTC as the example rather than 2AM

Reply:
Done